packages/teleterm/src/ui/services/modals/modalsService.ts

```diff
@@ -16,7 +16,8 @@ limitations under the License.
 
 import { useStore } from 'shared/libs/stores';
 
-import { ImmutableStore } from '../immutableStore';
+import { ImmutableStore } from 'teleterm/ui/services/immutableStore';
```

Comment:
> Wasn't `../` better here? I'd treat `services` as a single _module_, so it makes sense to me to use relative imports inside the module.

Hmm, I suppose I don't think of services as a single module, rather than a folder which groups things that fulfill a similar role, so as soon as I want to grab something that's outside of the folder of the specific "unit" (modalsService would be the "unit" in this case), I use a full path.

And I wanted to avoid having another import group (files from parent directories) because I felt that with parent imports it's hard to enforce how deep can you reach. Plus I'm only even interested in what stuff we import from our own package, which stuff we import from other Teleport packages and which stuff we import from built-in/3rd party packages.

Ultimately I don't really care about it as long as the computer can enforce our chosen import style, which I don't think we enforce in that scenario, so since I don't care I can bring back `../immutableStore` here. 😏

Reply:
I don't have any strong opinion on it too, it only seemed to me that in this case parent imports was fine :)